Data/send_mail.py

```diff
@@ -11,7 +11,7 @@
 smtp_port = 587
 
 # Load the Excel file
-file_path = 'Mail.xlsm'
+file_path = 'Data/Mail.xlsm'
```

Comment:
I don't think this is required. It will crash since the send_mail.py is already in the Data folder.